### PR TITLE
Limit height and weight values to 5000

### LINKED ISSUE #291: [PE-D][Tester B] Infinite Weight/Height?
Giving a person an arbitrarily high weight and height results in them having infinite weight/height.

Steps to recreate: set a person's weight/height to be arbitrarily high (eg 99999999999999999999999999999999) either through `edit` or `weight`.

Additionally, closing and opening Fitbook afterwards results in an error parsing storage.

-------------
Labels: `type.FunctionalityBug` `severity.Medium`
original: chiaryan/ped#8

### PR DESCRIPTION
**Pull request by ruijietay**
Resolves #291

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -142,6 +142,8 @@ Format: `help`
 Adds a client to the FitBook.
 
 Format: `add n/NAME p/PHONE_NUMBER [e/EMAIL] [a/ADDRESS] [w/WEIGHT] [h/HEIGHT] [nt/NOTE] [t/TAG]…​`
+* For `height` and `weight`, a reasonable maximum value of 5000 is allowed. Although some decimal values of 5000 are allowed (e.g. 5000.000001), the weight and height values will still rounded off to the nearest 1 decimal place (i.e. 5000.0).
+
```
Comment (ruijietay) [inline docs/UserGuide.md:146]:
```suggestion
* Refer to the list of valid parameters of each input for more details.
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -170,6 +172,7 @@ Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [w/WEIGHT] [h/HEIGH
 * The index **must be a positive integer** 1, 2, 3, …​
 * At least one of the optional fields must be provided.
 * Existing values will be updated to the input values.
+* For `height` and `weight`, a reasonable maximum value of 5000 is allowed. Although some decimal values of 5000 are allowed (e.g. 5000.000001), the weight and height values will still rounded off to the nearest 1 decimal place (i.e. 5000.0).
```
Comment (ruijietay) [inline docs/UserGuide.md:175]:
```suggestion
* Refer to the list of valid parameters of each input for more details.
```

File: docs/UserGuide.md [hunk 3]
```diff
@@ -251,6 +254,7 @@ Format: `weight INDEX [WEIGHT] `
 
 * Adds a weight value to a client specified by `INDEX`. The index refers to the index number shown in the displayed client list. The index **must be a positive integer** 1, 2, 3, …​
 * Weight values specified in this command will be added as a new weight value to the specified client. To edit the latest weight of the client, use the [`edit`](#editing-a-client--edit) command.
+* A reasonable maximum value of 5000 is allowed. Although some decimal values of 5000 are allowed (e.g. 5000.000001), the value will still rounded off to the nearest 1 decimal place (i.e. 5000.0).
```
Comment (ruijietay) [inline docs/UserGuide.md:257]:
```suggestion
* Refer to the list of valid parameters of each input for more details.
```

File: src/main/java/seedu/address/model/person/Height.java [hunk 4]
```diff
@@ -9,8 +9,9 @@
  * Guarantees: immutable; is always valid.
  */
 public class Height extends Attribute<Float> {
+    public static final Float HEIGHT_MAX_VALUE = 5000f;
 
-    public static final String MESSAGE_CONSTRAINTS = "Height value can only be a positive number.";
+    public static final String MESSAGE_CONSTRAINTS = "Height value can only be a positive number less than 5000.";
```
Comment (ruijietay) [inline src/main/java/seedu/address/model/person/Height.java:14]:
```suggestion
    public static final String MESSAGE_CONSTRAINTS = "Height value can only be a number between 0 and 5000 (inclusive).";
```

File: src/main/java/seedu/address/model/person/weight/Weight.java [hunk 5]
```diff
@@ -10,8 +10,9 @@
  * Guarantees: immutable; is always valid.
  */
 public class Weight extends Attribute<Float> {
+    public static final Float WEIGHT_MAX_VALUE = 5000f;
 
-    public static final String MESSAGE_CONSTRAINTS = "Weight value can only be a positive number.";
+    public static final String MESSAGE_CONSTRAINTS = "Weight value can only be a positive number less than 5000.";
```
Comment (ruijietay) [inline src/main/java/seedu/address/model/person/weight/Weight.java:15]:
```suggestion
    public static final String MESSAGE_CONSTRAINTS = "Weight value can only be a number between 0 and 5000 (inclusive).";
```

**Review by dillontkh — APPROVED** [review]
LGTM!

**Review by jjchee77 — APPROVED** [review]
LGTM! 👍

**Review by RyanNgWH — APPROVED** [review]
Looks good

**Review by bgopi23 — APPROVED** [review]
LGTM

**Pull request merged**